email/parser.go

```diff
@@ -27,7 +27,9 @@ const (
 )
 
 var (
-	skylinkRE           = regexp.MustCompile("^.*://([a-z0-9]{55})|://.*\\..*/([a-zA-Z0-9-_]{46}).*$")
+	skylink64RE = regexp.MustCompile(".+?://.+?\\..+?/([a-zA-Z0-9-_]{46})")
+	skylink32RE = regexp.MustCompile(".+?://([a-zA-Z0-9-_]{55})")
```

Comment:
nit: Can we call these `extractSkylinkXXRE` and/or add docstings for them?

Reply:
updated